EIPS/eip-5792.md

```diff
@@ -1,390 +1,269 @@
-transaction nonce. It is not trivial for the dapp to find all signed transactions for a given nonce and account,
-especially for smart contract accounts which usually store the nonce in a contract storage slot. This happens more
-frequently with smart contract wallets, which usually use a third party relaying service and automatically re-broadcast
-transactions with higher gas prices.
-
-These methods also do not support sending multiple function calls related to a single action. For example, when swapping
-on Uniswap, the user must often call [EIP-20](./eip-20.md) `#approve` before calling the Uniswap router contract to
-swap. The dapp has to manage a complex multi-step asynchronous workflow to guide the user through sending a single swap.
-The ideal UX would be to bundle the approve call with the swap call, and abstract the underlying approve function call
-from the user.
-
-The current interface also does not work well for account abstracted wallets (e.g. [EIP-4337](./eip-4337.md)
-or [EIP-3074](./eip-3074.md)), which often involve a relaying service to sign and broadcast the transaction that
-triggers the function calls from the user's wallet. In these cases the actual transaction hash may not be known at the
-time of user signing, but must still be returned by `eth_sendTransaction`. The transaction hash returned
-by `eth_sendTransaction` in these cases is unlikely to be relevant to the transaction hash of the included transaction.
-The existing interface also provides no way to delay the resolution of the transaction hash, since it is used as the key
-of the transaction tracked by the dapp. Dapps often link to the block explorer for the returned transaction hash, but in
-these cases the transaction hash is wrong and the link will not work.
-
-Dapps need a better interface for sending batches of function calls from the user's wallet so they can interact with
-wallets without considering the differences between wallet implementations. These methods are backwards compatible with
-existing EOA wallets. EOA wallets may send bundles of function calls as individual transactions. The goal of the new
-interface is to be a more stable and flexible interface that enables a better user experience, while wallet
-implementations evolve over time.
+The current methods used to send transactions from the user wallet and check their status are `eth_sendTransaction` and `eth_getTransactionReceipt`.
+
+The current methods used to send transactions from the user wallet and check their status do not meet modern developer demands and cannot accomodate new transaction formats. Even the name–`eth_sendTransaction`–is an artifact of a time when nodes served as wallets. 
+
+Today, developers want to send batch transactions, which many smart accounts can support atomically. Developers also want to make use of features afforded by new transaction formats, like paymasters in [ERC-4337](./eip-4337.md) transactions. `eth_sendTransaction` offers no way to do these things. 
```

Comment:
It'd be a good though-experiment if this can also support EIP-3074

Reply:
Should definitely be able to!

Reply:
gm gm sers! Batched `AUTHCALL`s are definitely doable here. Happy to collaborate where I can as a rep for 3074.